tools/deviceGen/output/limits_members.h

```diff
@@ -0,0 +1,582 @@
+    constexpr static inline uint32_t MinMaxImageDimension2D = 16384;
+    constexpr static inline uint32_t MinMaxSSBOSize = 1073741820;
+    constexpr static inline uint16_t MaxMaxPushConstantsSize = 256;
+    constexpr static inline uint32_t MinMaxWorkgroupCount = 65535;
+    constexpr static inline uint32_t MinMaxWorkgroupInvocations = 256;
+    constexpr static inline int32_t MinSubPixelInterpolationOffsetBits = 4;
+    // constexpr static inline core::bitflag<asset::IImage::E_SAMPLE_COUNT_FLAGS> NoMSor4Samples = asset::IImage::E_SAMPLE_COUNT_FLAGS::ESCF_1_BIT|asset::IImage::E_SAMPLE_COUNT_FLAGS::ESCF_5_BIT;
+    constexpr static inline uint8_t MinMaxColorAttachments = 8; // ROADMAP 2024 and wide reports
+    // constexpr static inline uint32_t MinInlineUniformBlockSize = 256;
+    /*
+       Vulkan 1.0 Core
+    */
+    uint32_t maxImageDimension1D = MinMaxImageDimension2D;
+    uint32_t maxImageDimension2D = MinMaxImageDimension2D;
+    uint32_t maxImageDimension3D = 2048;
+    uint32_t maxImageDimensionCube = MinMaxImageDimension2D;
+    uint32_t maxImageArrayLayers = 2048;
+    uint32_t maxBufferViewTexels = 33554432;
+    uint32_t maxUBOSize = 65536;
+    uint32_t maxSSBOSize = MinMaxSSBOSize;
+    uint16_t maxPushConstantsSize = 128;
+    uint32_t maxMemoryAllocationCount = 4096;
+    uint32_t maxSamplerAllocationCount = 4000;
+    uint32_t bufferImageGranularity = 65536; // granularity, in bytes, at which buffer or linear image resources, and optimal image resources can be bound to adjacent offsets in the same allocation
+    // size_t sparseAddressSpaceSize = 0; // [TODO LATER] when we support sparse
+    // uint32_t maxBoundDescriptorSets = 4; // [DO NOT EXPOSE] we've kinda hardcoded the engine to 4 currently
+    uint32_t maxPerStageDescriptorSamplers = 16; // Descriptors with a type of IDescriptor::E_TYPE::ET_COMBINED_IMAGE_SAMPLER count against this limit
```

Comment:
I need stuff properly grouped and newlines between groups like in the old members